hnn_core/viz.py

```diff
@@ -322,7 +322,7 @@ def plot_cells(net, ax=None, show=True):
     markers = {'L5_pyramidal': '^', 'L2_pyramidal': '^',
                'L5_basket': 'x', 'L2_basket': 'x'}
 
-    for cell_type in net.cellname_list:
+    for cell_type in net.cell_types:
```

Comment:
Great job catching this! I usually don't notice the viz is broken until way later..